src/components/Profile.js

```diff
@@ -1,7 +1,19 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { selectAllMyMissions } from '../redux/missions/missionsSlice.js';
 
-const Profile = () => (
-  <div>Profile</div>
-);
+const Profile = () => {
+  const MissionData = useSelector(selectAllMyMissions);
+
+  return (
+    <div className="container flex flex-row">
+      <div className="flex flex-col">
+        {MissionData.map(({ mission_name }) => (
```

Comment:
could you please look at the instruction?          
                                          
### Render UI: My Profile section
- Compose two/three column layout and list ONLY the rockets/dragons reserved and missions joined by the user (as per design):
    -  Render a list of all joined missions (use filter()).